spec/controllers/submissions_controller_spec.rb

```diff
@@ -1325,6 +1460,60 @@
         end
       end
     end
+    describe 'When the file is a url file' do
+      context 'with a valid url file format' do
+        let(:files) { [file7] }
+        before :each do
+          assignment.update!(url_submit: true)
+        end
+        it 'should return the file type' do
+          submission_file = submission.submission_files.find_by(filename: file7.original_filename)
+          get_as instructor, :get_file, params: { course_id: course.id,
+                                                  id: submission.id,
+                                                  submission_file_id: submission_file.id }
+          expect(JSON.parse(response.body)['type']).to eq 'markusurl'
```

Comment:
if you add a `format: :json` to the `params` on the previous line, you can use `response.parsed_body` here instead, which is a bit cleaner